lively.freezer/src/resolvers/node.cjs

```diff
@@ -48,24 +46,29 @@ function isAlreadyResolved(url) {
   if (url.startsWith('file://') ||
       url.startsWith('https://') ||
       url.includes('jspm.dev') ||
+      url.includes('esm://') ||
       url.startsWith('node:')) return true;
 }
 
 function ensureFileFormat(url) {
   return url && url.startsWith('/') ? 'file://' + url : url;
 }
 
-// fixme: if we are bundling from a node.js script but targeting the browser, we need to properly resolve /npm- urls
 function resolveModuleId (moduleName, importer, context = 'node') {
-  if (moduleName.startsWith('esm://cache/')) return moduleName.replace('esm://cache/', 'https://jspm.dev/'); // for now
+  if (moduleName.startsWith('esm://cache/')) {
+    return moduleName;
+  }
   if (importer && importer.startsWith('https://jspm.dev/')) {
     if (moduleName.startsWith('/')) 'https://jspm.dev' + moduleName;
     if (moduleName.startsWith('https://jspm.dev')) return moduleName;
   }
+  if (importer && importer.startsWith('esm://cache/')) {
+    if (moduleName.startsWith('/')) 'esm://cache' + moduleName;
```

Comment:
I somehow missed this looking at this code previously, but shouldn't this be returned or at least assigned somewhere? The same applies in line 62?

Reply:
Yes, you are correct about this. Now with the changes of this branch, this code here is really no longer executed and can be removed, assuming we define all our esm imports via `'esm://cache/etc'` inside the code we bundle. This is because we keep resolving the esm imports to `esm_cache` urls, so the case that an importer is a `esm://cache` like id never appears.
Previously we would replace the `esm://` urls with `jspm://` analogs, leading us to ignore the cache entirely during bundling. That made the code above somewhat necessary.